stack.py

```diff
@@ -0,0 +1,59 @@
+from static_array import StaticArray
+
+
+class Stack:
+    def __init__(self, size):
+        self.size = size
+        self.array = StaticArray(size)
+        self.occupied_size = 0
+
+    def push(self, element):
+        if self.occupied_size == self.size:
```

Comment:
How can we push IF we've reached size limitation ? Should it be `!=` operator?

Reply:
fixed